cross/npth/Makefile

```diff
@@ -0,0 +1,16 @@
+PKG_NAME = npth
+PKG_VERS = 1.2
+PKG_EXT = tar.bz2
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = ftp://ftp.gnupg.org/gcrypt/$(PKG_NAME)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/gnutls
+
+HOMEPAGE = 
```

Comment:
I can't find an homepage for nPth. Should I add http://gnupg.org, or http://lwn.net/Articles/496268/?